azext_iot/central/commands_destination.py

```diff
@@ -0,0 +1,180 @@
+from azext_iot.central.models.enum import ApiVersion
+from azext_iot.central.models.v1_1_preview import (
+    DestinationV1_1_preview,
+    WebhookDestinationV1_1_preview,
+    AdxDestinationV1_1_preview,
+)
+
+
+def get_dataExport_destination(
+    cmd,
+    app_id: str,
+    destination_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> Union[
+    DestinationV1_1_preview, WebhookDestinationV1_1_preview, AdxDestinationV1_1_preview
+]:
+    provider = CentralDestinationProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    return provider.get_dataExport_destination(
+        destination_id=destination_id, central_dnx_suffix=central_dns_suffix
+    )
+
+
+def delete_dataExport_destination(
+    cmd,
+    app_id: str,
+    destination_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+):
+    provider = CentralDestinationProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    provider.delete_dataExport_destination(
+        destination_id=destination_id, central_dnx_suffix=central_dns_suffix
+    )
+
+
+def list_dataExport_destinations(
+    cmd,
+    app_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> List[
+    Union[
+        DestinationV1_1_preview,
+        WebhookDestinationV1_1_preview,
+        AdxDestinationV1_1_preview,
+    ]
+]:
+    provider = CentralDestinationProvider(
+        cmd=cmd, app_id=app_id, api_version=api_version, token=token
+    )
+
+    return provider.list_dataExport_destinations(central_dns_suffix=central_dns_suffix)
+
+
+def add_dataExport_destination(
+    cmd,
+    app_id: str,
+    destination_id: str,
+    type,
+    display_name,
+    url=None,
+    cluster_url=None,
+    database=None,
+    table=None,
+    header_customizations=None,
+    authorization=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> Union[
+    DestinationV1_1_preview, WebhookDestinationV1_1_preview, AdxDestinationV1_1_preview
+]:
+    destination = {
+        "id": destination_id,
+        "type": type,
+        "displayName": display_name,
+    }
+
+    if type == DestinationType.Webhook.value:
+        if not url:
+            raise CLIError(
+                "url parameter is required when creating webhook destination."
+            )
+        destination.update({"url": url})
+        if header_customizations is not None:
+            destination.update(
+                {
+                    "headerCustomizations": utility.process_json_arg(
+                        header_customizations, argument_name="header"
+                    )
+                }
+            )
+
+    if type == DestinationType.AzureDataExplorer.value:
+        if not cluster_url:
```

Comment:
You can combine this error message into one: if one of the required parameters is missing throw "The parameters cluster-url, database, and table are required when creating an Azure Data Explorer destination

Reply:
More clear for customer now which parameter is missing, I would keep in this way.